### PR TITLE
✨ amp-story-player: Add play, pause and togglePaused methods to player

### PR DESCRIPTION
**Pull request by proyectoramirez**
This PR introduces 3 methods in the API of the amp-story-player: _play_, _pause_, and _togglePaused_. This allows users to programmatically play/pause the content and the auto-advance of the current playing story.

Partial for #28416

### CODE REVIEW

File: src/amp-story-player/amp-story-player-impl.js [hunk 1]
```diff
@@ -144,6 +144,9 @@ export class AmpStoryPlayer {
   attachCallbacksToElement_() {
     this.element_.load = this.load.bind(this);
     this.element_.show = this.show.bind(this);
+    this.element_.play = this.play.bind(this);
+    this.element_.pause = this.pause.bind(this);
+    this.element_.togglePaused = this.togglePaused.bind(this);
```
Comment (Enriqe) [inline src/amp-story-player/amp-story-player-impl.js:149]: Let's keep the API simple and intuitive and just have `play()` and `pause()` for now.

File: src/amp-story-player/amp-story-player-impl.js [hunk 2]
```diff
@@ -156,6 +159,37 @@ export class AmpStoryPlayer {
...
+  /**
+   * Makes the current story pause its content/auto-advance
+   */
+  pause() {
+    this.togglePaused(true);
+  }
+
+  /**
+   * Makes the current story play or pause its content/auto-advance
+   *
```
Comment (Enriqe) [inline src/amp-story-player/amp-story-player-impl.js:177]:
```suggestion
```

File: src/amp-story-player/amp-story-player-impl.js [hunk 3]
```diff
@@ -156,6 +159,37 @@ export class AmpStoryPlayer {
...
+   * @param {boolean} paused If true, the story will be paused, and it will be played otherwise
+   */
+  togglePaused(paused) {
```
Comment (Enriqe) [inline src/amp-story-player/amp-story-player-impl.js:180]: lets make this one private

File: src/amp-story-player/amp-story-player-impl.js (hunk 1 above)
Comment (proyectoramirez) [reply src/amp-story-player/amp-story-player-impl.js]: Done in latest commit

File: src/amp-story-player/amp-story-player-impl.js (hunk 3 above)
Comment (proyectoramirez) [reply src/amp-story-player/amp-story-player-impl.js]: Done!

File: src/amp-story-player/amp-story-player-impl.js (hunk 2 above)
Comment (proyectoramirez) [reply src/amp-story-player/amp-story-player-impl.js:178]: Done in latest commit!

Comment (Enriqe) [reply src/amp-story-player/amp-story-player-impl.js:178]: I still see it?

**Comment by Enriqe** [comment]
Seems like we need to resolve the conflicts again 😬 let me know when it's ready!

**Comment by Enriqe** [comment]
👏👏

**Pull request merged**